brave/browser-laptop
# Adds progress indicator to the import process

Submitter Checklist:


Resolves #9476

Auditors:

Test Plan:
- try to import data from firefox and check if your toggles are respected
- try to import big file (check qa resources for 5k file) and check if you see new text notifying you about import status
![image](https://user-images.githubusercontent.com/9574457/30533422-978be662-9c59-11e7-992e-472bbf6466aa.png)



Reviewer Checklist:

Tests
